src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -405,10 +384,94 @@ SynthesizedClosureEnvironment MakeFrame(Analysis.Scope scope, bool isStruct)
                     syntax,
                     methodId,
                     closureId);
+            }
         }
-    }
 
-        private SynthesizedClosureEnvironment GetStaticFrame(DiagnosticBag diagnostics, IBoundLambdaOrFunction lambda)
+        /// <summary>
+        /// Synthesize the final signature for all closures.
+        /// </summary>
+        private void SynthesizeLoweredFunctionMethods()
+        {
+            Analysis.VisitClosures(_analysis.ScopeTree, (scope, closure) =>
+            {
+                var originalMethod = closure.OriginalMethodSymbol;
+                var syntax = originalMethod.DeclaringSyntaxReferences[0].GetSyntax();
+                var structClosures = closure.CapturedEnvironments
+                    .Where(env => env.IsStruct).Select(env => env.SynthesizedEnvironment).AsImmutable();
+
```

Comment:
(Yes, I know this is just moved existing code)

---
In reply to: [133294526](https://github.com/dotnet/roslyn/pull/21408#discussion_r133294526) [](ancestors = 133294526)